enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -125,6 +143,15 @@ type Provider struct {
 }
 
 func NewProvider(env environment.Env, buildRoot string) (*Provider, error) {
+	// Unpack embedded execserver to buildRoot.
```

Comment:
Gate behind the flag?

Reply:
I can add the flag guard later - for now this makes it easier to test, since we can turn on this optimization by setting the `debug-use-exec-server` prop even if the flag is not enabled for all actions.